pkg/apis/autoscaling/v1alpha1/metricsproducer_defaults.go

```diff
@@ -44,7 +44,7 @@ func (s *ReservedCapacitySpec) defaultValues() {
 // Default ScheduleSpec
 func (s *ScheduleSpec) defaultValues() {
 	if s.Timezone == nil {
-		s.Timezone = ptr.String("UTC")
+		s.Timezone = aws.String("UTC")
```

Comment:
Please continue to rely on knative.pkg here. We want want an AWS dependency for this part of the code. At some point, the cloud provider will be removed and put in another repo.

Reply:
Ah I see. I was changing it here in respect to Prateek's comment, but didn't realize that `aws.String` was only used in cloud-provider specific packages. Thanks for the catch.